test/wolt_modal_sheet_test.dart

```diff
@@ -318,4 +320,27 @@ void main() {
       expect(modal.isAtLastPage, isTrue);
     });
   });
+
+  testWidgets('WoltModalSheet.modalBarrierColor defaults', (tester) async {
+    Color barrierColor = Colors.black54;
+    await tester.pumpWidget(buildSheetWithShow());
+
+    await tester.tap(find.text('Open sheet'));
+    await tester.pumpAndSettle();
+
+    final ColoredBox coloredBox = tester.widget(find.byType(ColoredBox));
```

Comment:
Why are we finding by ColoredBox to verify barrier?

Reply:
`ModalBarrier` in the` ModalRoute` uses the `ColoredBox` to render the barrier color and it's the only ColoredBox found in the test with expect value.

Reply:
What happens when the children also use ColoredBox let's say for example the drag handle. Which one does the test framework pick? You are finding by only type here.

Reply:
> What happens when the children also use ColoredBox let's say for example the drag handle. Which one does the test framework pick? You are finding by only type here.

The test simply launches the dialog here. We're not passing any additional `ColoredBox` in this isolated test. The test is correctly expecting one `ColoreBox here` with the barrier color.

Reply:
Now I seeç Thanks for clarıfıcatıon